ts/src/pro/bybit.ts

```diff
@@ -411,7 +411,7 @@ export default class bybit extends bybitRest {
         //     }
         //
         return [
-            this.safeInteger (ohlcv, 'timestamp'),
+            this.safeInteger (ohlcv, 'start'),
```

Comment:
@sc0Vu You sure `start` fits better here? 
![image](https://github.com/ccxt/ccxt/assets/43336371/c855f853-85b7-43e3-af99-cd807ff89c42)

Reply:
As I tested ccxt in binance  uses start as timestamp it is also a lot easier to combine multiple ohlcv datas to chart when you know exact start time of the candle. Bybit api returns that candle ohlcv data from start, not from timestamp.

Reply:
@augasur fair enough 🙂